config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do 
+  get 'likes/index'
```

Comment:
The rails generator probably added line 2 for you to the `routes.rb` file. This isn't needed - your likes routes are handled on line 13. 

Try to review all the changes that the generators make to your app (ideally immediately after the generator runs and before `git add`-ing) as they will sometimes add the wrong thing unfortunately! To review, use `git status` and `git diff` and `git diff --staged` from the command line.